README.md

```diff
@@ -1 +1,58 @@
-# s3same
\ No newline at end of file
+# s3same
+`s3same` ("sesame", like ["open sesame"](https://en.wikipedia.org/wiki/Open_Sesame_(phrase))) creates unique-per-repo AWS credentials for Travis CI artifact uploading to S3 and encrypts those credentials with the repo's public key.
+
+## Installation
+
+Running the command
+```sh
+sudo pip install git+https://github.com/vokal/s3same.git
+```
+will install `s3same` (from the master branch of the source of this repo) as a command in `/usr/local/bin`.
+
+## Usage
+
+```
+$ s3same --help
+Usage: s3same [OPTIONS]
+
+Options:
+  --repo TEXT         The target repository  [required]
+  --pro               Use Travis CI Pro
+  --github TEXT       GitHub token
+  --owner TEXT        GitHub owner
+  --s3-bucket TEXT    S3 bucket for artifacts
+  --aws-region TEXT   AWS region
+  --aws-key TEXT      AWS key
+  --aws-secret TEXT   AWS secret
+  --aws-profile TEXT  AWS profile
+  --help              Show this message and exit.
+```
+
+Let's assume you've got your AWS credentials in `~/.aws/credentials`.  Go to your GitHub settings and [create a personal access token](https://github.com/settings/tokens/new) for `s3same` (the token must have the `repo` permission).  Create `~/.s3same` with:
```

Comment:
I remember hearing about this somewhere but don't know what should be in `~/.aws/credentials`. Can this link to another doc for more details?